src/Time/getDateString.js

```diff
@@ -37,18 +49,19 @@ function getDateString({dateTime, locale, systemOffset = 0}) {
 	// A few seconds ago
 	if (offset < TIME_MINUTE / 2) {
 		delay = TIME_MINUTE / 2 - offset;
-		dateString = 'seconds ago';
+		const seconds = parseInt(offset / 1000);
+		dateString = secondsAgoReadout(seconds);
 	}
 	// Less than a minute ago
 	else if (offset < TIME_MINUTE) {
 		delay = TIME_MINUTE - offset;
-		dateString = '< 1 minute ago';
+		dateString = minutesAgoReadout(0);
 	}
```

Comment:
I'm wondering if we should get rid of this state and just keep going with seconds until the first minute is reached, or alternatively generously round up to a full minute?

Because the way translation libraries usually handle plurals without further customisation means that this will result in the readout "0 minutes ago".

If you want to keep this state, I'd give it its own custom readout getter, to make it very obvious that this third state exists (e.g. `lessThanOneMinuteAgoReadout`).

Reply:
The problem is that it can be a little jarring to update regularly. Perhaps we should just pass in the seconds and not show the count?

Reply:
@diondiondion i've changed this so that it will call `secondsAgoReadout` when under a minute. I've set the delay period to <=5 seconds. And set 0 seconds to be 1 second.

So starting from 0 we should expect to see something like "1 second", "5 seconds", "10 seconds", "15 seconds", .... "55 seconds", "1 minutes", "2 minutes", ... "9 minutes", "8:21 pm"

I'm tempted to jump in 10 second intervals. Can play around with it all we want and forever. 😄

Reply:
> @diondiondion i've changed this so that it will call `secondsAgoReadout` when under a minute. I've set the delay period to <=5 seconds. And set 0 seconds to be 1 second.
> 
> So starting from 0 we should expect to see something like "1 second ago", "5 seconds ago", "10 seconds ago", "15 seconds ago", .... "55 seconds ago", "1 minute ago", "2 minutes ago", ... "9 minutes ago", "8:21 pm"
> 
> I'm tempted to jump in 10 second intervals. Can play around with it all we want and forever. 😄

That sounds like a good solution. 10 seconds would've probably been fine, but let's just see how it feels in use.